Sources/SwifterSwift/CoreGraphics/CGVectorExtensions.swift

```diff
@@ -34,10 +34,9 @@ public extension CGVector {
 
     /// SwifterSwift: Creates a vector with the given magnitude and angle.
     ///
-    /// https://www.grc.nasa.gov/WWW/K-12/airplane/vectpart.html
-    ///
     ///     let vector = CGVector(angle: .pi, magnitude: 1)
     ///
+    /// - Note: https://www.grc.nasa.gov/WWW/K-12/airplane/vectpart.html
```

Comment:
I feel like this is more of a `See also` than a `Note`, since the note itself doesn't say anything other than the link.
```suggestion
    /// - See also: [Vector components](https://www.grc.nasa.gov/WWW/K-12/airplane/vectpart.html)
```